backend/src/prompts/templates/intent-system.j2

```diff
@@ -13,4 +13,7 @@ Output your result in the following json format:
     "question": "string of the original question",
     "user_intent": "string of the intent of the user's question",
     "questions": array of singular objective questions or if the question mentions csv, dataset or database an empty array
-}
\ No newline at end of file
+}
+
+Guidelines:
+- If the user has asked to check online, then each question in the questions array should also specify that.
```

Comment:
Why do we want this?
If this is a beneficial change, then I would expect more tests to be added to intent_config.yaml to prove that this is working as expected.

Reply:
I saw that Intent was not coming right as it for ESG related tasks, it was going to Datastore agent when there were more than 1 question. 
Sorry, I had no clue about intent_config.yaml, will have a look at it and add more tests related to it there.

Reply:
Do you have an example of when it was wrong and how, I'm still not quite getting the problem